tests/message/test_tasks.py

```diff
@@ -89,27 +89,6 @@ def test_task_to_compute_basic(self):
         msg = shortcuts.load(serialized, None, None)
         self.assertIsInstance(msg, message.tasks.TaskToCompute)
 
-    def test_task_to_compute_validate_compute_task_def(self):
```

Comment:
@Wiezzel well, I wouldn't remove the _whole_ test here - just remove the last check that tries to force the requestor_id to be different than the task owner...

Reply:
@shadeofblue Well, I removed the _whole_ method, so I don't see any reason for keeping this test case. What would it be?

Reply:
@Wiezzel okay, I was under impression that this test does a bit more than just test that single method but you may be right that that's not the case... though, slight decrease of coverage would suggest that it indeed does... but yes, that might not be an issue here...